modules/utils/filters.js

```diff
@@ -0,0 +1,13 @@
+import sanitizeHtmlFunction from 'sanitize-html';
+
+export const limitTo = (text, length) => {
+  return text.substring(0, length);
+};
+
+export const plainTextLength = (text) => {
+  return text && typeof(text) === 'string' ? String(text).replace(/&nbsp;/g, ' ').replace(/<[^>]+>/gm, '').trim().length : 0; // eslint-disable-line angular/typecheck-string
+};
+
+export const sanitizeHtml = (text) => {
+  return sanitizeHtmlFunction(text);
```

Comment:
This would need to be identical to this:

https://github.com/Trustroots/trustroots/blob/master/modules/offers/server/controllers/offers.server.controller.js#L78

(i.e. `sanitizeOptions`)